README.md

```diff
@@ -1,7 +1,14 @@
 # Sample code to interact with Amazon HealthLake service.
 
+You can see specific instructions under each folder. 
+
+## Repo Organization
+
+- hl-BATCH-operations folder contain samples in different lanagues (for now in Python) to perform batch operations. Think of a batch operation as creating multiple patient resources in Amazon HealthLake from a file (ndjson with multiple records).
```

Comment:
1. Make it "contains" instead of "contain"
2. You can remove the "(for now in Python)" part from this sentence.

Reply:
done